app/views/zendesk/tickets/show.html.erb

```diff
@@ -0,0 +1,84 @@
+<% content_for :page_title, "#{@ticket.subject}, ##{@ticket.id}" %>
+<% content_for :toolbar_right do %>
+  <div class="toolbar__item">
+    <span class="user__welcome">
+      Welcome
+      <strong class="user__name">
+        <%= current_user.name %>
+      </strong>
+    </span>
+  </div>
+<% end %>
+
+<% content_for :main do %>
+  <section class="slab slab--white steps--wrapper">
+    <div class="grid">
+      <div class="grid__item width-three-fourths shift-one-eighth">
+        <main role="main">
+          <div class="form-card">
+            <h1 class="form-question">
+              <%= @ticket.subject %>,
+              <%= link_to "##{@ticket.id}", "https://eitc.zendesk.com/agent/tickets/#{@ticket.id}" %>
+            </h1>
+
+            <div class="document-overview-list">
+              <%= image_tag "paper.svg", alt: "", class: "document-overview-list__icon" %>
+              <h2 class="h3 document-overview-list__heading">13614-C Intake Form</h2>
+              <ul class="document-overview-list__filenames">
+                <% @intakes.each do |intake| %>
+                  <li class="document-listing">
+                    <%= link_to pdf_zendesk_intake_path(intake.id, filename: "13614c_#{intake.name_for_filename}.pdf") do %>
+                      <%= "13614c_#{intake.name_for_filename}.pdf" %>
```

Comment:
Non-blocking, but because it's common for people to re-start the app, sometimes more than once, many tickets have an duplicate intake that doesn't have a 13614c. Not sure how that will behave here.

Reply:
It would show a mostly empty 13614-C